crt_portal/crt_portal/urls.py

```diff
@@ -25,4 +26,5 @@
     path('admin/', admin.site.urls),
     path('form/', include('cts_forms.urls')),
     path('report/', CRTReportWizard.as_view([WhatHappened, Where, Who, Details, Contact]), name='crt_report_form'),
+    path(r'', RedirectView.as_view(pattern_name='crt_report_form', permanent=False)),
```

Comment:
This redirect is a great addition, thank you @LindsayYoung! 

Tiny thing — I was curious about what the `r` was doing in `r''`, so I looked it up: https://stackoverflow.com/questions/33729045/what-does-an-r-represent-before-a-string-in-python?lq=1.

Turns out the `r` has to do with character escaping and tells Python to treat a string as a "raw string." I don't think we necessarily need it in this instance, but I also don't think it's important to take it out.

Reply:
I can take it out, no problem.